### PR TITLE
Update es.po

### PR DESCRIPTION
**Pull request by Mad-Soft**
Minor corrections based on Diablo 2 and grammar corrections.
I detect a error/crash. When start game for first time, go to Adria. Buy items and scroll down... and crash before of final list

### CODE REVIEW

File: Translations/es.po [hunk 1]
```diff
@@ -1,13 +1,17 @@
 # Translation of DevilutionX to Spanish
 # FIRST AUTHOR hiperiondev, 2021.
 # egonzalez, 2021.
+# ephphatha, 2021.
+# glebm, 2021.
+# Jmgr, 2021.
 # Mad-Soft <mad.soft@gmail.com>, 2021.
+# qndel, 2021.
```
Comment (AJenbo) [inline Translations/es.po:8]:
```suggestion
# Mad-Soft <mad.soft@gmail.com>, 2021.
```

**Comment by AJenbo** [comment]
> I detect a error/crash. When start game for first time, go to Adria. Buy items and scroll down... and crash before of final list

This is a known issue with the Spanish translation because of this line being incorrectly translated to expect additional parameters: `to hit: {:+d}%, {:+d}% damage`

**Comment by Mad-Soft** [comment]
> > I detect a error/crash. When start game for first time, go to Adria. Buy items and scroll down... and crash before of final list
> 
> This is a known issue with the Spanish translation because of this line being incorrectly translated to expect additional parameters: `to hit: {:+d}%, {:+d}% damage`
> 
> I had disabled the line to prevent the crash ([bf1b60f#diff-03ccae5a0dd13422f5ba1ec9e5a40c699f93e47eff75cfada31e28c124e70e72](https://github.com/diasurgical/devilutionX/commit/bf1b60ff2273f1e056f4d6a4dc443e2aaf340f10#diff-03ccae5a0dd13422f5ba1ec9e5a40c699f93e47eff75cfada31e28c124e70e72)), but it looks like you reintroduced it without correcting it in your previous update: [a1e9214#diff-03ccae5a0dd13422f5ba1ec9e5a40c699f93e47eff75cfada31e28c124e70e72R3863](https://github.com/diasurgical/devilutionX/commit/a1e9214a89f1554e47f810de7708fe0f755a2495#diff-03ccae5a0dd13422f5ba1ec9e5a40c699f93e47eff75cfada31e28c124e70e72R3863)

Feel free to change that line
msgid "to hit: {:+d}%, {:+d}% damage"
msgstr "al golpear: {:+d}%, {:+d} daño"

**Comment by AJenbo** [comment]
Thanks

**Pull request merged**